tests/neg/i17738-toplevel-infix.scala

```diff
@@ -0,0 +1,18 @@
+// scalac: -explain
```

Comment:
I tried using `//> using option -explain` but it just wouldn't work for some reason? Am I missing something here? [This ticket](https://github.com/lampepfl/dotty/issues/18149) says it should just work :disappointed:

Reply:
The ticket says "should work" but means "ought to work", but someone has to make it work. (And use it in tests after that.) It works on Scala 2, but unfortunately this PR is not a good candidate for backporting.

Reply:
So I just got got by the English language (and not totally misunderstood the other ticket's meaning, that'd be embarrassing 🦄 )